endpoints/apiconfig.go

```diff
@@ -777,7 +796,16 @@ func validateField(v reflect.Value, t reflect.StructField) error {
 
 	isZero := v.Interface() == reflect.Zero(v.Type()).Interface()
 	if isZero && tag.required {
-		return fmt.Errorf("missing field %v", t.Name)
+		return fmt.Errorf("missing field %v in %v", t.Name, parent.Name())
+	}
+
+	if v.Kind() == reflect.Struct {
+		if v.CanAddr() && hasExportedFields(v) {
```

Comment:
`CanAddr` is necessary, because of the `v.Addr()` line below. It might panic otherwise as stated here https://golang.org/pkg/reflect/#Value.Addr